packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'tabs.dart';
+
+/// Custom theme for [TabBar].
+///
+/// Used to customize the colors of the label, and the size/shape of the
+/// indicator.
+class TabBarTheme extends Diagnosticable {
```

Comment:
When I spoke to @HansMuller, he said that the themes that are not `InheritedWidget`s should be simple classes, rather than `InheritedModel`s. I will leave this comment unresolved for when @HansMuller is added to this PR so we can get confirmation.

Reply:
That's right. Theme will be an InheritedModel and TabBarTheme will be an "aspect" of Theme.